content/docs/dvclive/live/log_artifact.md

```diff
@@ -36,9 +36,10 @@ with Live() as live:
 
 ## Description
 
-Uses `dvc add` to track `path` with DVC, generating a `{path}.dvc` file. When
-combined with [`save_dvc_exp=True`](/doc/dvclive#initialize-dvclive), it will
-ensure that `{path}.dvc` is included in the experiment.
+Uses `dvc add` to [track] `path` with DVC, saving it to the DVC
+<abbr>cache</abbr> and generating a `{path}.dvc` file. When combined with
+[`save_dvc_exp=True`](/doc/dvclive#initialize-dvclive), it will ensure that
```

Comment:
`... {path}.dvc is included in the experiment.` - what does it mean from the user perspective? (why it's important for them)?

Reply:
Should we mention it can be used to track a reference to the cached file in Git? Is it going too deep and we should drop it altogether?

Reply:
yep, but would it be an expected result anyways? yep, it's either I miss the importance (it well might be, sometimes I'm lost a bit and don't follow all the details - like with --allow-missing) or yes it can be dropped if it's an expected behavior for users.

Reply:
by expected I mean the part that starts with ` When combined with [`save_dvc_exp=True`](/doc/dvclive#initialize-dvclive), it will ensure that ...` primarily.

Reply:
Sorry, I get it now. Okay, let's drop it since we suggest `save_dvc_exp=True` everywhere and plan to make it default.

Reply:
Updated